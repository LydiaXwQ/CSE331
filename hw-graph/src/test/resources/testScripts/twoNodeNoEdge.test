# Tests for a graph with two nodes and no edges.
# Test that a graph can be created, two nodes can be added, and the
# nodes and edges can be printed out.

# Create a graph
CreateGraph graph1

# Add two nodes
AddNode graph1 n1
AddNode graph1 n2

# Print the nodes in the graph and the outgoing edges from n1 and n2
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2