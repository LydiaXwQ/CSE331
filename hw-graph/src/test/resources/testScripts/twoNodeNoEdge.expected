# Tests for a graph with two nodes and no edges.
# Test that a graph can be created, two nodes can be added, and the
# nodes and edges can be printed out.

# Create a graph
created graph graph1

# Add two nodes
added node n1 to graph 1
added node n2 to graph 1

# Print the nodes in the graph and the outgoing edges from n1 and n2
graph1 contains: n1 n2
the children of n1 in graph1 are:
the children of n2 in graph1 are: